observations/forward_operators/obs_def_metar_mod.f90

```diff
@@ -6,7 +6,7 @@
 ! BEGIN DART PREPROCESS TYPE DEFINITIONS
 ! METAR_U_10_METER_WIND,           QTY_U_WIND_COMPONENT,       COMMON_CODE
 ! METAR_V_10_METER_WIND,           QTY_V_WIND_COMPONENT,       COMMON_CODE
-! METAR_TEMPERATURE_2_METER,       QTY_TEMPERATURE,            COMMON_CODE
```

Comment:
this is potentially going to affect other models.

Reply:
Alternatively, it is possible to impose a fix to the WRF model_mod.f90 directly,  leveraging the surface_var designation to force the operation on a surface temperature variable.

Reply:
I went with this alternative approach to model_mod.

Reply:
revert to main so we don't have obs_def_metar_mod.f90 whitespace changes in this pull request.